integration-tests/tests/specs/features/annotations.dependent.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { expect, test } from '@playwright/test';
+import { SearchPage } from '../../pages/search.page';
+
+test.describe('Sequence Preview Annotations', () => {
+    let searchPage: SearchPage;
+
+    test.beforeEach(async ({ page }) => {
+        searchPage = new SearchPage(page);
+    });
+
+    test.only('should have an embl file in the Files section', async ({ page }) => {
+        await searchPage.ebolaSudan();
+
+        const loculus_id = await searchPage.clickOnSequence(0);
```

Comment:
How do you know the first sequence is going to remain the same one? That seems like a too strong assumption that will break as soon as:
- a newer sequence gets submitted
- or we change sort order (which shouldn't affect thsi test here)

Reply:
i've changed the test to not use the pre-created sequence at all.